src/store/transactions/deposit/deposit.thunks.ts

```diff
@@ -157,7 +164,7 @@ function deposit(amount: BigNumber, ethereumAccount: EthereumAccount): AppThunk
     if (wallet !== undefined && signer !== undefined) {
       return adapters.hermezApi
         .deposit(
-          HermezCompressedAmount.compressAmount(amount.toString()),
+          amount,
```

Comment:
Why change this?

Reply:
This was a bug. We were already compressing the amount in the [deposit function of the hermez-api adapter](https://github.com/hermeznetwork/wallet-ui/blob/develop/src/adapters/hermez-api.ts#L602).